website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-gitlab.md

```diff
@@ -7,71 +7,112 @@ sidebar_label: "Connecting GitLab"
 ## Overview
 
 Connecting your GitLab account to dbt Cloud unlocks exciting and compelling functionality in dbt Cloud. Once your GitLab account is connected, you can:
-- [Trigger CI builds](cloud-enabling-continuous-integration-with-github) when Merge Requests are opened in GitLab.
-- Import new repos with one click
-- Carry GitLab user permissions through to dbt Cloud IDE's git actions.
-## For Developer and Team tiers
+- Log into dbt Cloud via OAuth through GitLab.
+- Import new GitLab repos with a couple clicks during dbt Cloud project setup.
+- Clone repos using HTTPS rather than SSH.
+- Trigger [Continuous integration](cloud-enabling-continuous-integration-with-github) builds when merge requests are opened in GitLab.
```

Comment:
is there a reason this user permissions bullet got dropped?
```suggestion
- Carry GitLab user permissions through to dbt Cloud IDE's git actions.
- Trigger [Continuous integration](cloud-enabling-continuous-integration-with-github) builds when merge requests are opened in GitLab.
```

Reply:
I need to specify "Carry GitLab user permissions through to dbt Cloud IDE's git actions." is only for the enterprise plan. We should delete the "log into dbt Cloud via OAuth through GitLab" altogether. That's incorrect.